src/routes/business/authorizedPerson.ts

```diff
@@ -0,0 +1,79 @@
+import type { Response } from "express";
+
+import generateID from "../../helpers/id";
+import uuid from "node-uuid";
+import { AuthorizedPerson } from "../../helpers/types";
+import { saveBusiness, getBusinessIdByAccount } from "../../db";
+import { RequestWithBusinessAndAccount } from "../../helpers/middlewares";
+
+export const createAuthorizedPerson = async (
+  req: RequestWithBusinessAndAccount,
+  res: Response
+) => {
+  const { business } = req;
+  const { account_id } = req.params;
+
+  const authorizedPersonId = req.body.authorized_person_id;
+  if (!authorizedPersonId) {
+    return res.status(400).send({
+      errors: [
+        {
+          id: uuid.v4(),
+          status: 400,
+          code: "missing_field",
+          title: "Missing Field",
+          detail: "Invalid request.",
+        },
+      ],
+    });
+  }
+
+  const businessId = await getBusinessIdByAccount({ id: account_id });
+  if (businessId !== business.id) {
+    return res.status(403).send({
+      errors: [
+        {
+          id: uuid.v4(),
+          status: 403,
+          code: "forbidden",
+          title: "Forbidden",
+          detail: "You are not authorized to perform this action.",
+        },
+      ],
+    });
+  }
+
+  try {
+    const authorizedPerson: AuthorizedPerson = {
+      id: generateID(),
+      authorized_person_id: authorizedPersonId,
+      business_id: business.id,
+      account_id,
```

Comment:
due to docs there is no business_id and account_id in response

Reply:
fixed on https://github.com/kontist/mock-solaris/pull/210/commits/38a6ae055df5f6ba12e246f2dc5ccc759deb5c64